runtime/rasdump/dmpmap.c

```diff
@@ -479,6 +499,57 @@ mapDumpSwitches(J9JavaVM *vm, J9RASdumpOption agentOpts[], IDATA *agentNum)
 
 	return OMR_ERROR_NONE;
 }
+
+/**
+ * Master disable heap, java, snap, and system dumps on out of memory
+ * @param vm Java vm
+ * @param @agentOpts dump agent options
+ * @param agentNum number of agents
+ */
+void
+disableDumpOnOutOfMemoryError(J9JavaVM *vm, J9RASdumpOption agentOpts[], IDATA agentNum)
+{
+	UDATA kindCursor = 0;
+
+	for (kindCursor = 0; kindCursor < numOomSettings; ++kindCursor) {
+		char *typeString = (char *) oomDefaultTable[kindCursor].typeString;
+		IDATA disableKind = scanDumpType(&typeString);
+		IDATA j = 0;
+		for (j = 0; j < agentNum; j++) {
+			if ((NULL != agentOpts[j].args) 
+					&& (agentOpts[j].kind == disableKind) 
+					&& (strcmp(agentOpts[j].args, oomDefaultTable[kindCursor].onOpt) == 0)) {
+				agentOpts[j].kind = J9RAS_DUMP_OPT_DISABLED;
+			}
+		}
+	}
+}
+
+/**
+ * Master enable heap, java, snap, and system dumps on out of memory
+ * @param vm Java vm
```

Comment:
This parameter is unused: why is it included?

Reply:
Removed.